charts/node-analyzer/README.md

```diff
@@ -40,7 +40,8 @@ helm install sysdig-agent --namespace sysdig-agent --create-namespace \
 --set nodeAnalyzer.secure.vulnerabilityManagement.newEngineOnly=true \
 --set global.kspm.deploy=true \
 --set nodeAnalyzer.nodeAnalyzer.benchmarkRunner.deploy=false \
---set nodeAnalyzer.nodeAnalyzer.hostScanner.deploy=true
+--set nodeAnalyzer.nodeAnalyzer.hostScanner.deploy=true \
+--set nodeAnalyzer.nodeAnalyzer.sslVerifyCertificate=false \
```

Comment:
From https://sysdig.atlassian.net/browse/DOC-3689, shouldn't the sslVerifyCertificate be added to the table below, and not to the example?

But the value is there. Is it just missing the default value:
![image](https://github.com/user-attachments/assets/4391803b-2fbe-44dd-8c45-e6fb252e29c6)

Reply:
What is the default value? I thought we need to add SSL verification to the example

Reply:
Not sure what the DOC ticket is asking about :) 

So adding to the example is ok. But the default value is missing. The default value AFAIK is "true" (verify certificate by default), or maybe it is empty achieving the same effect.